dex/src/main/scala/com/wavesplatform/dex/model/OrderBook.scala

```diff
@@ -12,7 +12,13 @@ import com.wavesplatform.dex.settings.MatchingRule
 
 import scala.collection.immutable.{HashMap, Queue, TreeMap}
 
-case class OrderBook private (bids: Side, asks: Side, lastTrade: Option[LastTrade], orderIds: HashMap[Order.Id, (OrderType, Price)]) {
+case class OrderBook private (
+  bids: Side,
+  asks: Side,
+  lastTrade: Option[LastTrade],
+  orderIds: HashMap[Order.Id, (OrderType, Price)],
+  nextTxTimestamp: Long
```

Comment:
lastTxTimestamp?

Reply:
it is an increased timestamp, so for next tx you will just use it without modifying. So it's nextTxTimestamp)